assets/queries/dockerfile/copy_instead_of_add/query.rego

```diff
@@ -0,0 +1,21 @@
+package Cx
+
+CxPolicy [ result ] {
+  resource := input.document[i].command[j]
+  check(resource, input.document[i].command[j+1]) == true
```

Comment:
We should change the logic of this index access, to ensure we always have a next element. Check the length of the array before and compare with the current index